Source/Particles/ParticleBoundaryBuffer.cpp

```diff
@@ -151,7 +151,7 @@ struct CopyAndTimestamp {
             dst.m_runtime_idata[j][dst_i] = src.m_runtime_idata[j][src_i];
         }
         dst.m_runtime_idata[m_step_index][dst_i] = m_step;
-        dst.m_runtime_rdata[m_time_index][dst_i] = m_step*m_dt;
+        dst.m_runtime_idata[m_delta_index][dst_i] = 0*m_dt; //delta_fraction is putted as zero, we need to calculate it
```

Comment:
```suggestion
        dst.m_runtime_idata[m_delta_index][dst_i] = 0._rt; //delta_fraction is initialized to zero
```
Just checking . if its set to 0., we dont need to multiple with `m_dt`? 
also this is where its getting initialized and will be computed later in `FinedEmbeddedBoundaryIntersection`

Reply:
Yes exactly, for now we don't have yet the deltaTimeScraped for regular boundaries. We just assume that it is equal to 0 temporarily. I just did 0*dt to remember that we have to calculate the dt_fraction for the regular boundaries. 
Yes, we can just put it equal to 0 for now!
Thank you for the review Revathi!!